packages/react/src/errorboundary.tsx

```diff
@@ -1,9 +1,12 @@
 import { captureException, ReportDialogOptions, Scope, showReportDialog, withScope } from '@sentry/browser';
-import { logger, parseSemver } from '@sentry/utils';
+import { logger } from '@sentry/utils';
 import hoistNonReactStatics from 'hoist-non-react-statics';
 import * as React from 'react';
 
-const reactVersion = parseSemver(React.version);
+export function isAtLeastReact17(version: string): boolean {
```

Comment:
slightly tangential but since 'what we export' is often a topic of discussion currently, what do we think of exporting only for the sake of testing?